pkg/descheduler/strategies/nodeutilization/lownodeutilization.go

```diff
@@ -50,30 +50,33 @@ func LowNodeUtilization(ctx context.Context, client clientset.Interface, strateg
 	if strategy.Params != nil {
 		nodeFit = strategy.Params.NodeFit
 	}
-
+	useDeviationThresholds := strategy.Params.NodeResourceUtilizationThresholds.UseDeviationThresholds
 	thresholds := strategy.Params.NodeResourceUtilizationThresholds.Thresholds
 	targetThresholds := strategy.Params.NodeResourceUtilizationThresholds.TargetThresholds
-	if err := validateLowUtilizationStrategyConfig(thresholds, targetThresholds); err != nil {
+	if err := validateLowUtilizationStrategyConfig(thresholds, targetThresholds, useDeviationThresholds); err != nil {
 		klog.ErrorS(err, "LowNodeUtilization config is not valid")
 		return
 	}
-	// check if Pods/CPU/Mem are set, if not, set them to 100
-	if _, ok := thresholds[v1.ResourcePods]; !ok {
-		thresholds[v1.ResourcePods] = MaxResourcePercentage
-		targetThresholds[v1.ResourcePods] = MaxResourcePercentage
-	}
-	if _, ok := thresholds[v1.ResourceCPU]; !ok {
-		thresholds[v1.ResourceCPU] = MaxResourcePercentage
-		targetThresholds[v1.ResourceCPU] = MaxResourcePercentage
-	}
-	if _, ok := thresholds[v1.ResourceMemory]; !ok {
-		thresholds[v1.ResourceMemory] = MaxResourcePercentage
-		targetThresholds[v1.ResourceMemory] = MaxResourcePercentage
+	// for % deviations, we'll handle that in the getNodeUsage function
+	if !useDeviationThresholds {
```

Comment:
`thresholds` and `targetThresholds` have to be set for all three CPU, memory and pods resources in every case. The default value in the `useDeviationThresholds` case is 0 (`MinResourcePercentage`). Which will help to transform:
```
if _, ok := lowThreshold[v1.ResourcePods]; !ok {
	lowResourceThreshold[v1.ResourcePods] = nodeCapacity.Pods()
	highResourceThreshold[v1.ResourcePods] = nodeCapacity.Pods()
} else {
	lowResourceThreshold[v1.ResourcePods] = resource.NewQuantity(int64(normalizeResourcePercentage(averagePodsUsagePercent-float64(lowThreshold[v1.ResourcePods]))*float64(nodeCapacity.Pods().Value())*0.01), resource.DecimalSI)
	highResourceThreshold[v1.ResourcePods] = resource.NewQuantity(int64(normalizeResourcePercentage(averagePodsUsagePercent+float64(highThreshold[v1.ResourcePods]))*float64(nodeCapacity.Pods().Value())*0.01), resource.DecimalSI)
}
```
into
```
if lowThreshold[v1.ResourcePods] == MinResourcePercentage {
	lowResourceThreshold[v1.ResourcePods] = nodeCapacity.Pods()
	highResourceThreshold[v1.ResourcePods] = nodeCapacity.Pods()
} else {
	lowResourceThreshold[v1.ResourcePods] = resource.NewQuantity(int64(normalizeResourcePercentage(averagePodsUsagePercent-float64(lowThreshold[v1.ResourcePods]))*float64(nodeCapacity.Pods().Value())*0.01), resource.DecimalSI)
	highResourceThreshold[v1.ResourcePods] = resource.NewQuantity(int64(normalizeResourcePercentage(averagePodsUsagePercent+float64(highThreshold[v1.ResourcePods]))*float64(nodeCapacity.Pods().Value())*0.01), resource.DecimalSI)
}
```